content/docs/command-reference/version.md

```diff
@@ -12,17 +12,17 @@ usage: dvc version [-h] [-q | -v]
 
 `dvc version` outputs the following information about the system/environment:
 
-| Line                                        | Detail                                                                                                                                                               |
-| ------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| [`DVC version`](#components-of-dvc-version) | Version of DVC, and name of the binary or package manager (`pip`, `conda`, etc.) used to install DVC (along with a Git commit hash in case of a development version) |
-| `Platform`                                  | Version of Python in the environment where DVC is initialized, and information about the operating system                                                            |
-| `Supports`                                  | Types of [remote storage](/doc/command-reference/remote/add#supported-storage-types) supported by the current DVC setup (their required dependencies are installed)  |
-| `Cache types`                               | [Types of links](/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache) supported (between <abbr>workspace</abbr> and <abbr>cache</abbr>)     |
-| `Cache directory`                           | Filesystem type (e.g. ext4, FAT, etc.) and drive on which the <abbr>cache</abbr> directory is mounted                                                                |
-| `Caches`                                    | Cache [location types](/doc/user-guide/managing-external-data) configured in the repo (e.g. local, SSH, S3, etc.)                                                    |
-| `Remotes`                                   | Remote [location types](/doc/command-reference/remote/add#supported-storage-types) configured in the repo (e.g. SSH, S3, Google Drive, etc.)                         |
-| `Workspace directory`                       | Filesystem type (e.g. ext4, FAT, etc.) and drive on which the <abbr>workspace</abbr> is mounted                                                                      |
-| `Repo`                                      | Shows whether we are in a DVC repo and/or Git repo                                                                                                                   |
+| Line                                        | Detail                                                                                                                                                                           |
+| ------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
These crazy long changes are due to linting: MD tables need to accommodate for the length of the updated links in every row...